noms_ops.ini

```diff
@@ -40,3 +40,4 @@ spooler = %d/spooler
 spooler-chdir = %d
 spooler-import = mtp_%n/tasks.py
 cron = 13 8 -1 -1 1 %d/venv/bin/python %d/manage.py check_notify_templates --verbosity 2
+cron = 7 -2 -1 -1 -1 %d/venv/bin/python %d/manage.py load_locations_from_offender_search --verbosity 2
```

Comment:
in standard cron, this is `7 */2 * * *`
i.e. 7 min past the hour, every 2 hours

Reply:
i wanted it to be more frequent than now (manual; once daily)… if not 2 hours, then how often would you do it? ppl do get transferred around…

Reply:
Just thinking aloud really, don't have any strong opinions and I like the increased frequency and we can always reduce if it causes issues 👍